features/data/events/rdf/mixed-event-with-online-url-and-multiple-calendar.ttl

```diff
@@ -33,8 +33,8 @@
   generiek:naamruimte "http://data.uitdatabank.local:80/events/" ;
   generiek:lokaleIdentificator "%{eventId}" .
 
-<http://data.uitdatabank.local:80/events/%{eventId}#virtueleLocatie-61d7772d>
-  a platform:virtueleLocatie ;
+<http://data.uitdatabank.local:80/events/%{eventId}#virtualLocation-61d7772d>
+  a schema:virtualLocation ;
```

Comment:
@grubolsch this is missing a capitalised V for the Class Name

`a schema:VirtualLocation`